### ISSUE TITLE
Migrate ruby build to bazel

### ISSUE THREAD

**Issue by shs96c** [SeleniumHQ/selenium#7772]

**Comment by AutomatedTester** [comment]
Removing Selenium-4-Beta label as this is not strictly required to get to Selenium 4 and is gated on issues with Ruby and Bazel.

**Comment by diemol** [comment]
I believe this was already done by @p0deje, please reopen if I am wrong.
